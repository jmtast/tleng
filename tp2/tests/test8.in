// Error de sem√°ntica: 0 pulsos
#tempo negra 120
#compas 0/4

const oct1 = 5;

voz (0)
{
  compas
  {
  }
}
