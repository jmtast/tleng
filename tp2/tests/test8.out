Error:
Cantidad de pulsos incorrecta. Debe ser mayor a 0. LÃ­nea 3
